src/zzz_od/gui/view/hollow_zero/lost_void_setting_interface.py

```
@@ -151,13 +151,17 @@ def on_interface_shown(self) -> None:
 
     def _update_run_record_display(self) -> None:
         if self.run_record.bounty_commission_complete:
-            content = '已完成悬赏委托 如错误可重置'
+            content = gt('已完成悬赏委托 如错误可重置')
```

Ruff (0.16.1)

[error] Undefined name `gt`

(F821)